script/openqa-bootstrap-container

```diff
@@ -14,7 +14,13 @@ set -x
 CONTAINER_NAME="openqa1"
 CONTAINER_PATH="/var/lib/machines/${CONTAINER_NAME}"
 
-DEFAULT_REPO="${DEFAULT_REPO:="http://download.opensuse.org/tumbleweed/repo/oss/"}"
+ARCH="${ARCH:=$(arch)}"
+if [ "$ARCH" = "x86_64" ]; then
+    DEFAULT_REPO="${DEFAULT_REPO:="http://download.opensuse.org/tumbleweed/repo/oss/"}"
+else
+    DEFAULT_REPO="${DEFAULT_REPO:="http://download.opensuse.org/ports/$ARCH/tumbleweed/repo/oss/"}"
+fi
```

Comment:
I guess this triggered https://travis-ci.org/os-autoinst/openQA/jobs/556335220#L590 now. Probably by adding quotes in L65 you should be good to go.

Reply:
Done. But not sure why the other 2 tests are failing now.